### PR TITLE
feat: migrate Loader component

### LINKED ISSUE #19: Migrate Loader component

### PR DESCRIPTION
**Pull request by SeqviriouM**
Migrate Loader component

Resolves #19

### CODE REVIEW

**Comment by SeqviriouM** [comment]
@amje can u check PR pls

File: src/components/Loader/Loader.scss [hunk 1]
```diff
@@ -0,0 +1,102 @@
+@import '../variables';
+
+$block: '.#{$ns}label';
```
Comment (amje) [inline src/components/Loader/Loader.scss:3]: loader

File: src/components/Loader/Loader.scss [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    width: $height * 0.25;
+    height: $height;
+}
+
+@mixin loaderPseudoSize($height) {
+    height: calc(#{$height} / 1.5);
+    width: $height * 0.25;
+}
+
+.yc-loader {
```
Comment (amje) [inline src/components/Loader/Loader.scss:30]: should use `$block` variable

File: src/components/Loader/Loader.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+$active-color: var(--yc-color-loader-active);
+$pulse-duration: 800ms;
+$pulse-delay: $pulse-duration * 0.25;
+$animation: yc-pulse ease $pulse-duration infinite;
+
+$base-height-s: 20px;
+$base-height-m: 28px;
+$base-height-l: 36px;
+
+@keyframes yc-pulse {
```
Comment (amje) [inline src/components/Loader/Loader.scss:14]: use prefix `$ns` variable here

File: src/components/Loader/Loader.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    height: $height;
+}
+
+@mixin loaderPseudoSize($height) {
+    height: calc(#{$height} / 1.5);
+    width: $height * 0.25;
+}
+
+.yc-loader {
+    $block: &;
```
Comment (amje) [inline src/components/Loader/Loader.scss:31]: Already declared at the top

File: src/components/Loader/Loader.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import block from 'bem-cn-lite';
```
Comment (amje) [inline src/components/Loader/Loader.tsx:2]: Instead raw lib use `block` from `src/components/utils/cn.ts`, that create class with prefix

File: src/components/Loader/index.ts [hunk 6]
```diff
@@ -0,0 +1 @@
+export * from './Loader';
```
Comment (amje) [inline src/components/Loader/index.ts:1]: reexport from main index also needed

File: src/components/Loader/__stories__/Loader.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+} as Meta;
+
+const DefaultTemplate: Story<LoaderProps> = (args) => <Loader {...args} />;
+export const Default = DefaultTemplate.bind({});
+
+const SizeTemplate: Story<LoaderProps> = (args: any) => {
+    return (
+        <>
+            <Loader {...args} size="s">
+                s
```
Comment (amje) [inline src/components/Loader/__stories__/Loader.stories.tsx:17]: Does Loader take children?

File: src/components/Loader/__stories__/Loader.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
```
Comment (amje) [inline src/components/Loader/__stories__/Loader.stories.tsx:2]: `types-6-0` part can be removed

File: src/components/Loader/__stories__/Loader.stories.tsx [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+export default {
+    title: 'Components/Loader',
+    component: Loader,
+} as Meta;
+
+const DefaultTemplate: Story<LoaderProps> = (args) => <Loader {...args} />;
+export const Default = DefaultTemplate.bind({});
+
+const SizeTemplate: Story<LoaderProps> = (args: any) => {
```
Comment (amje) [inline src/components/Loader/__stories__/Loader.stories.tsx:13]: Here args should be infered as LoaderProps

File: src/components/Loader/README.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+Loader
+---------------
+
+Component for rendering loading state (flashing bars)
+
+## Properties
+| Property     | Type       | Required | Default | Description |
+|:---          |:---        |:---:     |:---     |:---         |
+| size         | `String`   |          | 'm'     |Component size. Available values: `s`, `m`, `l` |
+| className    | `String`   |          |         |CSS class   |
```
Comment (amje) [inline src/components/Loader/README.md:10]: Let's rewrite this into TS interface with JSDoc comments. See Button.md for reference

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Loader/__stories__/Loader.stories.tsx (hunk 9 above)
Comment (SeqviriouM) [reply src/components/Loader/__stories__/Loader.stories.tsx:13]: Yeap, any is unnecessary

File: src/components/Loader/__stories__/Loader.stories.tsx (hunk 7 above)
Comment (SeqviriouM) [reply src/components/Loader/__stories__/Loader.stories.tsx:17]: Nope, thanks for your attention

**Review by amje — APPROVED** [review]

**Pull request merged**